lib/cmd/common/initlocal.js

```diff
@@ -16,7 +16,7 @@ var fs = require('fs');
 var url = require('url');
 var exec = require('child_process').exec;
 var cheerio = require('cheerio');
-var env = require('./env.js');
+var _ = require('underscore');
```

Comment:
I'd imagine nobody is using `initlocal` any more, but just incase, here's some backwards compatibility. Fixing the references was easier than ripping the command...